src/elements/content-preview/preview-header/PreviewHeader.js

```diff
@@ -100,7 +101,7 @@ const PreviewHeader = ({
                                     {...contentOpenWithProps}
                                 />
                             )}
-                            {shouldRenderAnswers && <ContentAnswers />}
+                            {shouldRenderAnswers && <ContentAnswers fileName={fileName} />}
```

Comment:
we should just pass the `file` into the component, we need like 5 props from the file and passing them one by one.  Over here, you can see that I started to do that https://github.com/box/box-ui-elements/pull/3418/files#diff-66b69814f9065a2b0cf810889871d57679e2bbb4ace5991907c86085d75c3b29R18